spec/controllers/application_controller_spec.rb

```diff
@@ -1369,4 +1369,45 @@ def hub_path; end
       end
     end
   end
+
+  describe "#ensure_non_admins_reset_passwords_when_forced" do
+    let(:non_admin_user) { create(:team_member_user) }
+    let(:admin_user) { create(:admin_user) }
+
+    context "when a non-admin is logged in" do
+      before do
```

Comment:
Code style blocker:

We should move the `let` into this block. It's a common pattern in our code to have the `let` _within_ the `context... when`. Let's stick to it unless there's a reason it doesn't fit here; happy to chat.

Unless we need both users to exist in the database at once, I recommend renaming to `let(:user)` within the block.

There's one more way to make these tests more concise (and concise is good). Since the code doesn't refer to the `user` except in the `before`, it's an option not to really have a variable:

```
    context "when a non-admin is logged in" do
      before { sign_in(create(:admin_user, forced_password_reset_at: nil)) }

      it "redirects to change the password" do 
      # ...
      end
    end
```

The only essential change is moving the `let`s into the `context` where they are relevant. I do also suggest, non-blockingly, any other changes like this that can make the tests more concise.